src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/advice/CaseDocumentControllerAdvice.java

```diff
@@ -92,6 +93,11 @@
             ErrorConstants.NOT_FOUND.getErrorCode(), ErrorConstants.NOT_FOUND.getErrorMessage());
     }
 
+    @ExceptionHandler(HttpClientErrorException.class)
+    protected ResponseEntity<Object> handleHttpClientErrorException(final HttpClientErrorException exception) {
+        return new ResponseEntity<>(exception.getResponseBodyAsString(), exception.getStatusCode());
+    }
+
```

Comment:
Can we change structure of `ErrorResponse` as well  as discussed earlier? Or are you going create an another PR ?